### ISSUE TITLE
Goodreads requirements change

### ISSUE THREAD

**Issue by epsilon-0** [janeczku/calibre-web#1351]
The current good reads plugin is out of date and has been archived by upstream  
  - https://github.com/sefakilic/goodreads/  

It has been succeeded by the betterreads library: 
  - https://github.com/thejessleigh/betterreads

It would be good to see this change reflected here and see if it is compatible still.

**Comment by OzzieIsaacs** [comment]
Thank you for the hint, it's indeed better to rely on a supported project than on a abandoned. 
Upon trying to integrate it, it turned out that it is not working. 
I created a pull request for the necessary fixes, let's see how it evolves: https://github.com/thejessleigh/betterreads/pull/37.
Some changes in Calibre-Web are required, as the API of betterreads changed compared to goodreads. In the development tree I already made the necessary changes.
